aspnetcore/fundamentals/minimal-apis/includes/parameter-binding8-10.md

```diff
@@ -186,6 +186,8 @@ The following code shows the complete `Program.cs` file:
 
 #### File uploads using IFormFile and IFormFileCollection
 
+File uploads using `IFormFile` and `IFormFileCollection` in minimal APIs require `multipart/form-data` encoding. The parameter name in the route handler must match the form field name in the request. Minimal APIs don't support binding the entire request body directly to an `IFormFile` parameter without form encoding.
```

Comment:
Can we refer readers to the section(s) that describe how to bind the entire request body without form encoding?